services/commands-service.ts

```diff
@@ -295,6 +304,16 @@ export class CommandsService implements ICommandsService {
 		}).future<boolean>()();
 	}
 
+	private getCommandHelp(): IFuture<any> {
+		return (() => {
+			if (_.isEmpty(this.cachedCommandHelp) && this.$fs.exists(this.$resources.resolvePath(this.$staticConfig.COMMAND_HELP_FILE_NAME)).wait()) {
```

Comment:
this.cachedCommandHelp should be null, and the isEmpty check is redundant. You can simply check for existence.